### ISSUE TITLE
Flash tinkerboard image on tinkerboard s

### ISSUE THREAD

**Issue by plchampigny** [TinkerBoard/debian_kernel#34]
I've tried to flash a tinkerboard image (2.0.7) on a tinkerboard s. The image has been read using win32diskimager from a modified version of the tinkerboard (only apps have been installed). If I flash this image on the eMMC or on a sd card, the tinkerboard s does not boot (or at least doesnt send anything to the hdmi). If I flash that image on an sd card and put it in a 1st gen tinkerboard, it boots up properly.

Is there something I have to do to fix my image for tinkerboard s?

**Comment by jamess-huang** [comment]
Hi, 
   Did you use the same peripheral between tinker and tinker S ? (HDMI, sdcard ... and so on)
   If not, please use the same peripheral as tinker and try again.
   If yes, may I know your adaptor SPEC ? 5V/2A or 5V/3A ?

**Comment by plchampigny** [comment]
I do believe that my AC adapter is a 2.5A/5V. I'll check tomorrow at work. Meanwhile I can tell you I plugged the same sd card that is working on a tinkerboard on a tinkerboard s and I used the same ac, hdmi, etc and its not working.

**Comment by jamess-huang** [comment]
When the tinker S is un-boot, may I know the 3 LEDs status ? Is the red LED on and is the yellow LED blinking ?

**Comment by krystofurr** [comment]
I purchased a 1st gen Tinkerboard recently and tried to use the v2.07 image ( also tried 2.08 ).  Got a similiar result.  No boot.  Black screen using HDMI connection.  I only have a red LED on myself.

**Comment by jamess-huang** [comment]
Hi, krystofurr
    Can you see the yellow LED blink sometimes ?

**Comment by krystofurr** [comment]
Yes, occassionally they do.  I tried v1.8 and it seems to work fine.  Funny that the main website doesn't have the latest version for download.  I downloaded the image from here. https://www.asus.com/ca-en/Single-Board-Computer/Tinker-Board/

**Comment by jamess-huang** [comment]
Hi, krystofurr
    The case you met is difference from plchampigny's. Yours is a HDMI compatibility issue. May I know your HDMI model name ?

**Comment by krystofurr** [comment]
I'll try to find the exact models.  The first was a 32 inch Sony bravia lcd tv ( works fine with an rpi 3 and v1. 8 of the os).  The second was a 50 inch insignia led tv (Also works with rpi 3 and v1. 8 of the os). When it did come up with 1.8 though the resolution is very off and if u disconnect / reconnect the HDMI it fixes a little.  Also there is a high static noise that is constant. This only happens with the sony bravia

**Comment by krystofurr** [comment]
Got them.

Sony Bravia KDL-32ML130
Insignia NS-50L260A13

**Comment by jamess-huang** [comment]
Hi, krystofurr
    Have you tried the below method with v2.0.8 image ?
    https://tinkerboarding.co.uk/wiki/index.php?title=HDMI-resolution
